dashboard/app/api.go

```diff
@@ -139,26 +139,23 @@ func handleAPI(c context.Context, r *http.Request) (reply interface{}, err error
 	password := r.PostFormValue("key")
 	ns, err := checkClient(getConfig(c), client, password, subj)
 	if err != nil {
-		if client != "" {
-			log.Errorf(c, "checkClient('%s') error: %v", client, err)
-		} else {
-			// Don't log as error if somebody just invokes /api.
```

Comment:
Keep this comment where we check empty client.

Reply:
Done.